README.md

```diff
@@ -1,86 +1,49 @@
 
-![Cozy Logo](https://raw.github.com/cozy/cozy-setup/gh-pages/assets/images/happycloud.png)
+The [`cozy-desktop`](https://github.com/cozy-labs/cozy-desktop) repository is where we do development and there are many ways you can participate in the project, for example:
 
-[Cozy](https://cozy.io/) is a platform that brings all your web services in the same private
-space.  With it, your web apps and your devices can share data easily,
-providing you with a new experience. You can install Cozy on your own hardware
-where no one profiles you.
+-   [Submit bugs and feature requests](https://github.com/cozy-labs/cozy-desktop/issues) and help us verify as they are checked in.
+-   Review [source code changes](https://github.com/cozy-labs/cozy-desktop/pulls).
+-   Review the [documentation](https://github.com/cozy-labs/cozy-desktop/tree/master/doc) and make pull requests for anything from typos to new content.
 
+## Contributing
 
-## What is Cozy Drive for Desktop?
+If you are interested in fixing issues and contributing directly to the code base,
+please see the document [How to Contribute](https://github.com/cozy-labs/cozy-desktop/wiki/How-to-Contribute), which covers the following:
 
-The Cozy Drive for Desktop application allows to sync the files stored in your
-Cozy with your laptop and/or your desktop. It replicates your files on your
-hard drive and apply changes you made on them on other synced devices and on
-your online Cozy.
+-   [How to build and run from source](doc/developer/setup.md)
+-   [Other developer documentation](doc/developer/)
 
 ## Disclaimer
 
 :warning: Cozy Drive for Desktop is still in beta version. Backup your data before using it!
 
-
-## Installing & using the application
-
-See the [official documentation](https://docs.cozy.io/en/sync/desktop/).
-
-
-## Advanced use cases
-
-- [Installing and using the command-line version](doc/usage/cli.md)
-- [Synchronizing multiple directories](doc/usage/multi_directory_setup.md)
-- [Ignore files](doc/usage/ignore_files.md)
-- [Limitations](doc/usage/limitations.md)
-
-
-## Cozy v2 users
-
-The new Cozy Drive for Desktop application targets the new Cozy stack v3 only.
-
-Maintenance for the old cozy-desktop GUI / CLI (targeting Cozy v2) is still
-occuring here. Support requests are definitely welcome, although we're quite
-busy with the v3 release.
-
-
-## GNU/Linux & BSD users
-
-The new Cozy Drive for Desktop application should work on Linux and
-BSD flavors. It has been tested in development on Ubuntu/Gnome, however we do not provide init scripts, service files or packages yet, so you'll have to build and install it by hand. Any contribution in this direction will be appreciated.
-Packages for major GNU/Linux distributions should come up.
-
-
-## Contributing
-
-- [Setting up a development environment](doc/developer/setup.md)
-- [Other developer documentation](doc/developer/)
-
-
-## Get in touch
-
-The best way to ask for help is through the form embedded in the application.
```

Comment:
Synchronization issues are better reported through the help form included into the app (so logs are automatically joined to the request).